index.bs

```diff
@@ -12636,7 +12635,7 @@ internal method as follows.
     The algorithm operates as follows, with property name |P| and object |O|:
 
     1.  If |P| is not a [=supported property name=] of |O|, then return false.
-    1.  If |O| has an own property named |P|, then return false.
+    1.  If <a abstract-op>HasOwnProperty</a>(|O|, |P|) is <emu-val>true</emu-val>, then return false.
```

Comment:
Unfortunately, I think this is an infinite recursion.

Consider a call to [[GetOwnProperty]] as defined in <https://heycam.github.io/webidl/#legacy-platform-object-getownproperty>.  This calls https://heycam.github.io/webidl/#LegacyPlatformObjectGetOwnProperty which calls into https://heycam.github.io/webidl/#dfn-named-property-visibility for _O_.  The proposed change would then call https://tc39.github.io/ecma262/#sec-hasownproperty on _O_, which will call right back into _O_'s [[GetOwnProperty]].

What you probably need to do instead is invoke https://tc39.github.io/ecma262/#sec-ordinarygetownproperty directly or something.  But note that this uses exactly the same vague "has an own property" (or rather "does not have an own property") language that webidl already has...